lib/avo/execution_context.rb

```diff
@@ -4,6 +4,8 @@ class ExecutionContext
 
     attr_accessor :target, :context, :params, :view_context, :current_user, :request, :include, :main_app, :avo, :locale
 
+    delegate_missing_to :@view_context
```

Comment:
Isn't this cleaner? Is there a performance impact for doing it like this?

```suggestion
    delegate_missing_to :view_context
```

Reply:
I think there is a performance impact because `view_context` it's just:
```ruby
def view_context
  @view_context
end
```

So it's an extra unnecessary call

https://www.greyblake.com/blog/ruby-perfomance-tricks/#use-instance-variables

I'm surprised about this :point_up:  , never thought that's "about two times faster"

If is that much faster we should consider refactor and use the instance variable when available instead the reader method.

Reply:
Good to know! Let's keep that in mind!